packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -3413,6 +3327,18 @@ namespace Tpetra {
     indicesAreAllocated_ = true;
     indicesAreLocal_     = true;
     pftype_              = StaticProfile; // if the profile wasn't static before, it sure is now.
+
+    // Column indices sorted by row
```

Comment:
1. Were there actual cases where they were _not_ sorted by row on input?
2. MueLu usually wants to give Tpetra sorted arrays here -- should we add an option not to sort? 

@csiefer2

Reply:
That is a vestige of the last PR where indices are sorted and I didn’t mean for it to sneak through.  I’ll remove the changes so setAplIndices’ behavior is not changed

Reply:
Reverted to previous behavior in latest commit